src/client/_clientwindow.py

```diff
@@ -169,15 +173,16 @@ def __init__(self, *args, **kwargs):
         self.players = Playerset()  # Players known to the client
 
         self.gameset = Gameset(self.players)
-        fa.instance.gameset = self.gameset  # FIXME
+        self.modset = Modset()
+        fa.instance.gameset = self.gameset  # FIXME(needed fa/game_process L81 for self.game = self.gameset[uid])
```

Comment:
I wonder if it's me that put this here. Either way it's a really bad idea. If it's me, I'll look into why it's actually here.

Reply:
My comment mentioned where it is needed, and it's the only place (I checked).
It could be replaced with an import client in fa/game_process.py and then use client.instance.gameset[uid] in L81(I tested it, works).